packages/core/src/main/Account.ts

```diff
@@ -238,7 +238,7 @@ export class Account<T = any> extends EventEmitter {
     if (initClient) {
       await this.initClient({clientType});
 
-      if (this.mlsConfig) {
+      if (this.mlsConfig && this.backendFeatures.supportsMLS) {
```

Comment:
The only place I could move it would be the constructor, which does not satisfy typescript later in the methods (I would still need to check for `this.mlsConfig` since its optional param.

Reply:
This does not satisfy typescript unfortunately :/